lib/rules/require-shorthand-arrow-functions.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Require shorthand arrow function expressions when returning a single statement.
```

Comment:
Maybe i was the only one confused by this, but if you could clarify this a bit more, that would be great